spec/vehicle_registration_spec.rb

```diff
@@ -0,0 +1,17 @@
+require './lib/facility'
+require './lib/vehicle'
+require "./lib/vehicle_registration"
+
+RSpec.describe VehicleRegistration do
```

Comment:
Remember we want the test file to match the class file, so this file should be called `vehicle_spec.rb`